titanfall2updates.json

```diff
@@ -0,0 +1,84 @@
+        "UpdateSourceConditionSetting": "RemoveExcessFiles",
+        "UpdateSourceConditionValue": "true",
+        "SkipOnFailure": true
+    },
+    {
+        "UpdateStageName": "Remove Excess Files",
+        "UpdateSourcePlatform": "Windows",
+        "UpdateSource": "Executable",
+        "UpdateSourceData": "powershell.exe",
+        "UpdateSourceArgs": "-NoProfile -Command \"Set-Location -Path 'titanfall2\\1237970'; $ErrorActionPreference='SilentlyContinue'; Remove-Item -Recurse -Force vpk\\client_sp_*,vpk\\englishclient_sp_*,r2\\cfg\\*.ekv,bin\\x64_retail\\client.dll,bin\\x64_retail\\auditionwin64.dll,bin\\dxsupport.cfg,Titanfall2.exe,r2\\maps,r2\\media,r2\\screenshots,platform,Core,Support,__Installer; Get-ChildItem r2\\paks\\Win64 -File | Where-Object { $_.Name -notmatch '^patch_master\\.rpak$' -and $_.Name -notmatch '^common(_mp)?(\\(.*\\))?\\.rpak$' -and $_.Name -notlike 'ui*' } | Remove-Item -Force; Get-ChildItem r2\\sound -File | Where-Object { $_.Name -ne 'titanfall_2.mprj' } | Remove-Item -Force\"",
+        "UpdateSourceConditionSetting": "RemoveExcessFiles",
+        "UpdateSourceConditionValue": "true",
+        "SkipOnFailure": false
+    },
+    {
+        "UpdateStageName": "Northstar Download",
+        "UpdateSourcePlatform": "All",
+        "UpdateSource": "GithubRelease",
+        "UpdateSourceArgs": "R2Northstar/Northstar",
+        "UpdateSourceTarget": "{{$FullBaseDir}}",
+        "UnzipUpdateSource": true,
+        "OverwriteExistingFiles": true,
+        "DeleteAfterExtract": true,
+        "SkipOnFailure": false
+    },
+    {
+        "UpdateStageName": "Northstar Stubs Download",
+        "UpdateSourcePlatform": "Windows",
+        "UpdateSource": "GithubRelease",
+        "UpdateSourceArgs": "R2Northstar/NorthstarStubs",
+        "UpdateSourceData": "NorthstarStubs.zip",
+        "UpdateSourceTarget": "{{$FullBaseDir}}bin/x64_retail",
+        "UnzipUpdateSource": true,
+        "OverwriteExistingFiles": true,
+        "DeleteAfterExtract": true,
+        "SkipOnFailure": false
+    },
+    {
+        "UpdateStageName": "Northstar Stubs Download",
+        "UpdateSourcePlatform": "Linux",
+        "UpdateSource": "GithubRelease",
+        "UpdateSourceArgs": "R2Northstar/NorthstarStubs",
+        "UpdateSourceData": "NorthstarStubs.zip",
+        "UpdateSourceTarget": "{{$FullBaseDir}}",
+        "UnzipUpdateSource": true,
+        "OverwriteExistingFiles": true,
+        "DeleteAfterExtract": true,
+        "SkipOnFailure": false
+    },
+    {
+        "UpdateStageName": "Build NSWrap Wrapper",
+        "UpdateSourcePlatform": "Linux",
+        "UpdateSource": "Executable",
+        "UpdateSourceData": "/bin/bash",
+        "UpdateSourceArgs": "-c \"mkdir -p titanfall2/1237970/nswrap && cd titanfall2/1237970/nswrap && [[ -f nswrap.c ]] && rm -rf nswrap.c; wget -q https://raw.githubusercontent.com/pg9182/nsdockerwine2/refs/heads/master/nswrap/nswrap.c && gcc -Wall -Wextra nswrap.c -o nswrap >/dev/null\"",
```

Comment:
I strongly recommend pinning this to a hash or copying it somewhere, as that repository is my testing one and will change in incompatible ways.